modules/oc-mirror-updating-cluster-manifests.adoc

```diff
@@ -9,6 +9,8 @@
 
 After you have mirrored your image set to the mirror registry, you must apply the generated `ImageContentSourcePolicy`, `CatalogSource`, and release image signature resources into the cluster.
 
+If you set `graph:true` in your `ImageSetConfiguration` file, the oc-mirror plugin generates an `UpdateService.yaml` file. The `UpdateService` resource can be used to configure the OSUS service on the cluster side. See "About the OpenShift Update Service" for more information.
```

Comment:
Doesn't look like there's a matching `.Additional resources` after this module (at the assembly level) to provide the `xref` to the "About the OpenShift Update Service" section.